layout/theme.liquid

```diff
@@ -33,7 +33,7 @@
     <script src="{{ 'pubsub.js' | asset_url }}" defer="defer"></script>
     <script src="{{ 'global.js' | asset_url }}" defer="defer"></script>
     {%- if settings.animations_reveal_on_scroll -%}
-      <script src="{{ 'animations.js' | asset_url }}" id="AnimationsScript" defer="defer"></script>
+      <script src="{{ 'animations.js' | asset_url }}" defer="defer"></script>
```

Comment:
I was going to ask if you found significant improvements adding the script via snippet over this approach. Not sure I could tell.

Reply:
Actually not too sure, here is what the CI has given me: 
- `animation.js` approach: 77, 82, 81
- inline in `<script>`: 90, 78, 82, 89

But when testing in webpagetest.org the results are pretty similar. 🤷